<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
			background-color: #FFFFFF;
			border: 1px solid gray;
			color: #000000
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonClear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonChangeSign">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPercentage">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDivision">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMultiply">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSubstract">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAdd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonComma">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #D3D3D3;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEqual">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #00BFFF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSqrt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #1E90FF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSquare">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #1E90FF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>x² </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonInverse">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #1E90FF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCube">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #1E90FF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>x³</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackSpace">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
			background-color: #1E90FF;
			border: 1px solid gray;
			padding: 5px;
			color: #000000;
}
QPushButton:pressed{
			background-color: #A9A9A9;
			border: 1px solid gray;
			padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
